firmware/common/portapack_persistent_memory.cpp

```diff
@@ -200,6 +200,9 @@ void set_serial_format(const serial_format_t new_value) {
 	data->serial_format = new_value;
 }
 
+bool show_gui_return_icon(){ // add return icon in touchscreen menue
```

Comment:
Could you move this below the comment block , and include bit 20 in comment.

Reply:
Done